src/modules/blocks/ticket/container-content/attendee-collection/iac-setting/__tests__/template.test.js

```diff
@@ -7,7 +7,7 @@ import IACSetting from './../template';
 
 describe( 'IACSetting', () => {
 	// @todo @juanfra @rafsuntaskin fix this test from failing.
```

Comment:
Do we need to remove this todo?

Reply:
Nice catch @codingmusician , this comment was removed